.eslintrc

```diff
@@ -0,0 +1,20 @@
+---
+extends: react-app
```

Comment:
Correct. It uses the create-react-app built-in rules as a base so there won't be any conflicts with what the CRA linter is trying to do. Everything else is built on top of that. CRA starts from a VERY minimal base, so there's a lot of room to add rules on top, including whether lines should end with semicolons, what quotes to use, etc.